### PR TITLE
Bileworm post-merge feedback and qol

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

* Adds a death message to the bileworm!
  * some extra coder stuff about this: moved deathsound up from /mob to /living as it is a var for dying on a type that doesn't die, and moved deathmessage down from /simple_animal to /living as /simple_animal isn't where mobs learn to die! Also, they're snake case now, which is why this pr looks so big.
* Bileworms now give up their target if they attempt to resurface near them but cannot.
* Bileworms controlled by players can now see in the dark!

## Why It's Good For The Game

Just went around to various servers asking how people were feeling. From watching people fight bileworms, I'm really happy with how they turned out, but there is room for improvement!

## Changelog

:cl:
qol: Bileworms possessed by players can now see in the dark!
qol: Bileworms now have a death message!
balance: Bileworms will now give up their targeting if they fail to burrow to you. If they can still see you, they might regain aggro, but now moving say, to the mining station/big lava lake/hirophant is a good way to de-aggro them.
/:cl:

### CODE REVIEW

File: code/datums/actions/mobs/adjust_vision.dm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	icon_icon = 'icons/mob/actions/actions_animal.dmi'
+	button_icon_state = "adjust_vision"
+	background_icon_state = "bg_default"
+
+/datum/action/adjust_vision/Grant(mob/living/grant_to)
+	. = ..()
+	grant_to.see_in_dark = NIGHTVISION_FOV_RANGE
+	grant_to.lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+
+/datum/action/adjust_vision.Remove(mob/living/remove_from)
```
Comment (LemonInTheDark) [inline code/datums/actions/mobs/adjust_vision.dm:14]: arm wtf

File: code/datums/actions/mobs/adjust_vision.dm [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	. = ..()
+	grant_to.see_in_dark = initial(grant_to.see_in_dark)
```
Comment (LemonInTheDark) [inline code/datums/actions/mobs/adjust_vision.dm:16]: armmmmmmmm COMPILE

File: code/datums/actions/mobs/adjust_vision.dm [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	grant_to.lighting_alpha = initial(grant_to.lighting_alpha)
+
+/datum/action/adjust_vision/Trigger(trigger_flags)
+	. = ..()
+	if(!.)
+		return
+
+	var/mob/living/living_owner = owner
+	living_owner.sight = initial(sight_flags)
```
Comment (LemonInTheDark) [inline code/datums/actions/mobs/adjust_vision.dm:25]: AGHHHHHHHHHH

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Comment by tralezab** [comment]
uhhh dont peek at those resolved reviews

File: code/modules/mob/living/simple_animal/hostile/hivebot.dm [hunk 4]
```diff
@@ -40,7 +40,7 @@
 
 /mob/living/simple_animal/hostile/hivebot/Initialize(mapload)
 	. = ..()
-	deathmessage = "[src] blows apart!"
+	death_message = "[src] blows apart!"
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/simple_animal/hostile/hivebot.dm:43]: who the fuck set this on initialize

Comment (tralezab) [reply code/modules/mob/living/simple_animal/hostile/hivebot.dm:43]: LOL

**Pull request merged**